src/endpoints/tokens/token.service.ts

```diff
@@ -838,6 +838,21 @@ export class TokenService {
       token => token.transactions ?? 0,
     );
 
+    const egldIdentifier = 'EGLD-000000';
+    const egldToken = new TokenDetailed({
+      identifier: egldIdentifier,
+      name: 'EGLD',
+      type: TokenType.FungibleESDT,
+      assets: await this.assetsService.getTokenAssets(egldIdentifier),
+      decimals: 18,
+      isLowLiquidity: false,
+      price: await this.dataApiService.getEsdtTokenPrice(egldIdentifier),
```

Comment:
i think `egldIdentifier` should be replaced with `EGLD`

Reply:
fixed